aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -177,7 +221,12 @@ impl Aggregator {
             },
             AggregatorState::PositiveDelta => {
                 // If positive delta, add directly but also record the state.
-                self.value = addition(self.value, value, self.limit)?;
+                self.value = addition(self.value, value, self.limit).map_err(|err| {
+                    if self.value < u128::MAX - value {
```

Comment:
Maybe let's add a comment why we do not have to record if we overflow u128::MAX?

Reply:
Done.